Sources/Commands/SwiftTool.swift

```diff
@@ -366,6 +357,19 @@ public class SwiftTool<Options: ToolOptions> {
         }
     }
 
+    func buildPlan(graph: PackageGraph, config: Build.Configuration) throws -> BuildPlan {
```

Comment:
Side thought: would you also just `loadPackageGraph()` inside `buildPlan`? In that case, we wouldn't need any arguments and it could become a property.